README.md

```diff
@@ -33,6 +33,11 @@ For each field, you can select the field type from:
 For every field you can set a label and a help text.
 For select, radio and checkbox fields, you can select a list of values.
 
+## Captcha verification
+
+This form addon is configured to work with [HCaptcha](https://www.hcaptcha.com) and [ReCaptch](https://www.google.com/recaptcha/) to prevent spam.
```

Comment:
Typo: missing "a" at the end of "ReCaptch"

Reply:
Fixed in `main`